### PR TITLE
Script to replace problematic send_jsbox action

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: go/scripts/jsbox_send.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+    worker_creator = WorkerCreator(options.vumi_options)
+    worker_creator.create_worker_by_class(
+        JsBoxSendWorker, options.get_vumigo_config())
+
+    in_file = sys.stdin
+    out_file = sys.stdout if not quiet else None
+
+    worker = yield JsBoxSendWorker.WORKER_QUEUE.get()
+    yield worker.process_file(in_file, out_file)
```
Comment (hodgestar) [inline go/scripts/jsbox_send.py]: This is the only reference I can find to `.process_file` so I assume something else is supposed to be called instead?

Comment (jerith) [inline go/scripts/jsbox_send.py]: Whoops, that's left over from the `inject_messages` script I pilfered the structure from.

**Comment by hodgestar** [comment]
:+1:

**Comment by smn** [comment]
Build failing because of npm issues, restarting.

**Comment by smn** [comment]
:+1: from me trusting that there's no other thing we can wait on. Lip stick on pig feature would be to make it configurable, but it's still a pig after that.

**Pull request merged**